tools/hdf_converter/hdf_converter.xml

```diff
@@ -0,0 +1,29 @@
+<tool id="apCLMS_to_RamClustR_converter" name="apLCMS to RamClustR converter" version="0.0.1" python_template_version="3.5">
+    <description>converts aplcms HDF output to RamClustR csv input</description>
+    <requirements>
+        <container type="docker">maxskorik/hdf_converter</container>
```

Comment:
if we are going to use this in our resulting pipeline this should all go under recetox namespace, even on docker hub

Reply:
In fact, why do we have this container here? We just need Python and Pandas, right?

Reply:
Yes, just python and pandas. I thought there is no pandas in galaxy thus made a tool in a container. So is there pandas installed?

Reply:
It also needs python tables package to read HDF

Reply:
This is a Galaxy tool, it can define its `requirements` and Galaxy will resolve it, e.g. via Conda. Conda has pandas [here](https://anaconda.org/conda-forge/pandas). For checking what is available visit https://anaconda.org/ and search.